src/index.js

```diff
@@ -1,7 +1,14 @@
 const Stripe = require("stripe");
 
 async function handleRequest(request, env) {
-  const stripe = Stripe(env.STRIPE_API_KEY);
+  const stripe = Stripe(env.STRIPE_API_KEYY, {
+    apiVersion: '2023-08-16',
```

Comment:
```suggestion
  const stripe = Stripe(env.STRIPE_API_KEY, {
```
nit: typo fix + remove `apiVersion` since there's no need for it and we won't have to keep it up to date with future API versions. :)

Reply:
@anniel-stripe Thanks for quickly feedback!
I've fixed it.